testsuite/policy/authorization/auth_config.py

```diff
@@ -75,14 +73,20 @@ def remove_all_hosts(self):
         self.model.spec.hosts = []
 
     def wait_for_ready(self):
-        """Waits until authorization object reports ready status"""
-        with oc.timeout(90):
-            success, _, _ = self.self_selector().until_all(
-                success_func=lambda obj: len(obj.model.status.conditions) > 0
-                and all(x.status == "True" for x in obj.model.status.conditions)
-            )
-            assert success, f"{self.kind()} did not get ready in time"
-            self.refresh()
+        """Waits until authorization object reports Enforced=True status"""
+        success = self.wait_until(
+            lambda obj: all(x.status == "True" for x in obj.model.status.conditions)
+            and any(x.type == "Enforced" for x in obj.model.status.conditions),
+            90,
```

Comment:
90 is default, I would not explicitly repeat it here until we feel the 90 is really required here

Reply:
Ok, will refactor.

Reply:
Done